tests/evaluate/test_default_evaluator.py

```diff
@@ -3517,3 +3517,6 @@ def test_target_prediction_col_mapping():
                 "correctness/v1/variance": 0.0,
                 "correctness/v1/p90": 3.0,
             }
+
+
+# bbqiu add tests for recall
```

Comment:
shall we add this in this PR :) ?

Reply:
just getting things ready for bug bash -- tests will be added before merging haha